src/main/java/org/opensearch/security/dlic/rest/api/MultiTenancyConfigApiAction.java

```diff
@@ -177,6 +183,19 @@ private void updateAndValidatesValues(final ConfigV7 config, final JsonNode json
         if (Objects.nonNull(jsonContent.findValue(MULTITENANCY_ENABLED_JSON_PROPERTY))) {
             config.dynamic.kibana.multitenancy_enabled = jsonContent.findValue(MULTITENANCY_ENABLED_JSON_PROPERTY).asBoolean();
         }
+        if (Objects.nonNull(jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS))
+            && jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS).size() > 0) {
+            JsonNode newOptions = jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS);
```

Comment:
wdyt about taking advantage of streams?

i.e.

```
List<String> newOptions = jsonContent.findValuesAsText(DASHBOARD_SIGNIN_OPTIONS);
List<DashboardSignInOption> options = newOptions.stream().map(DashboardSignInOption::valueOf).collect(Collectors.toList());
```

Reply:
Sure, thanks for the feedback.

Reply:
@cwperks  I tried, however `findValuesAsText` returns empty.
Also, the returned values using `JsonNode newOptions = jsonContent.findValue(DASHBOARD_SIGNIN_OPTIONS);` are json nodes, it needs to access to each node by its index, so I had to go back to the previous version.